
<launch>

	<arg name="scan_topic" default="/scan"/>
	<arg name="odometry_topic" default="/vesc/odom"/>
	<arg name="laser_ray_step" default="18"/>
	<arg name="max_particles" default="4000"/>
	<arg name="motion_model" default="kinematic" />
	<arg name="max_range_meters" default="5.6" />
	<arg name="resample_type" default="naiive" />
	<arg name="bag_file" default="$(find lab1)/bags/loop.bag"/>

	<node pkg="lab1" type="OpenLoopRollout.py" name="open_loop_rollout" output="screen">
		<param name="scan_topic" value="$(arg scan_topic)"/>
		<param name="odometry_topic" value="$(arg odometry_topic)"/>
		<param name="laser_ray_step" value="$(arg laser_ray_step)"/>
		<param name="max_particles" value="$(arg max_particles)"/>
		<param name="motion_model" value="$(arg motion_model)" />
		<param name="max_range_meters" value="$(arg max_range_meters)" />
		<param name="resample_type" value="$(arg resample_type)" />
		<param name="max_viz_particles" value="60"/>
		<param name="bag_file" value="$(arg bag_file)"/>
	</node>
</launch>
